### PR TITLE
feat(logger): Support logger v2

### PR DESCRIPTION
**Pull request by jchauncey**
This PR submits the necessary code changes to support the new v2 logger implementation.

### CODE REVIEW

File: client/controller/models/apps/apps.go [hunk 1]
```diff
@@ -92,7 +92,7 @@ func Logs(c *client.Client, appID string, lines int) (string, error) {
 	u := fmt.Sprintf("/v2/apps/%s/logs", appID)
 
 	if lines > 0 {
-		u += "?log_lines=" + strconv.Itoa(lines)
+		u += "/?log_lines=" + strconv.Itoa(lines)
```
Comment (krancour) [inline client/controller/models/apps/apps.go]: Do logger URLs absolutely need a slash at the end now?  It seems a generally accepted practice that the final trailing slash in a URL is irrelevant.

Comment (jchauncey) [inline client/controller/models/apps/apps.go]: yeah for some reason I had to add the slash in the client to get it to talk to the logger component. I'll look into seeing if I can fix this.

Comment (krancour) [inline client/controller/models/apps/apps.go]: Sorry... I read this wrong.  This is in client code.  Maybe this has to do with recent Django updates?

File: rootfs/api/models.py [hunk 2]
```diff
@@ -475,7 +475,7 @@ def _default_scale(self, user, release):
     def logs(self, log_lines=str(settings.LOG_LINES)):
         """Return aggregated log data for this application."""
         try:
-            url = "http://{}:{}/{}?log_lines={}".format(settings.LOGGER_HOST, settings.LOGGER_PORT,
+            url = "http://{}:{}/{}/?log_lines={}".format(settings.LOGGER_HOST, settings.LOGGER_PORT,
```
Comment (krancour) [inline rootfs/api/models.py]: This is the spot I thought I was commenting on before.  I checked and the extra slash isn't needed here.  But it also doesn't hurt, so I won't make a fuss. :)

Comment (jchauncey) [inline rootfs/api/models.py]: Yeah I've already made the slash optional but haven't pushed that yet

**Comment by jchauncey** [comment]
Ok so this PR is ready to start having serious reviews. 

* You will need to build a new deis client: `cd client && make build`
* You will need to use the [deis-logger chart](https://github.com/deis/charts/pull/27)
* You will also need to build a custom workflow image and deploy it: `make docker-build docker-push kube-update`

Go through the steps to create and deploy an application. Then you should be able to use `deis logs` like normal.

If you have any questions let me know.

**Pull request merged**